konflux-ci/ui/core/proxy/proxy.yaml

```diff
@@ -186,6 +186,13 @@ spec:
           - https://localhost:9443/oauth2/callback
           - --oidc-issuer-url
           - https://localhost:9443/idp/
+          - --skip-oidc-discovery
```

Comment:
so oauth2-proxy won't try to connect to the external IP for discovery during deployment (when the service UI is exposed via public IP, i.e. `https://<public-ip>:9443`), otherwise the deployment fails